css/css-text/white-space/break-spaces-newline-011.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Text: 'white-space: break-spaces' with a sequence of white spaces and 1 line feed</title>
+
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
+  <link rel="help" href="https://www.w3.org/TR/css-text-3/#white-space-property">
+  <link rel="match" href="../../reference/ref-filled-green-100px-square.xht">
+  <link rel="stylesheet" type="text/css" href="/fonts/ahem.css">
+
+  <meta content="" name="flags">
+  <meta content="This test checks that when 'white-space' is set to 'break-spaces', then line feeds (&amp;NewLine; in the code) are preserved, just like with 'white-space: pre' or with 'white-space: pre-wrap'.">
+
+  <style>
+  div
+    {
+      font-family: Ahem;
+      font-size: 25px;
+      line-height: 1;
+      width: 4em;
+    }
+
+  div#overlapped-red-reference
+    {
+      background-color: red;
+      height: 4em;
+    }
+
+  div#overlapping-green-test
+    {
+      background-color: green;
+      bottom: 4em;
+      color: red;
+      position: relative;
+      white-space: break-spaces;
+    }
+  </style>
+
+  <p>Test passes if there is a filled green square and <strong>no red</strong>.
+
+  <div id="overlapped-red-reference"></div>
+
+  <div id="overlapping-green-test">             &NewLine;</div>
```

Comment:
I am confused. The test renders the same if the `&Newline;` is removed…

Reply:
I was confused. A line feed is not printable character. This 011 test will be removed. The other 012-016 tests, once corrected, will be useful and achieving their goals.